pylsp/workspace.py

```diff
@@ -389,6 +389,12 @@ def jedi_script(self, position=None, use_document_path=False):
             jedi.settings.auto_import_modules = jedi_settings.get('auto_import_modules',
                                                                   DEFAULT_AUTO_IMPORT_MODULES)
             environment_path = jedi_settings.get('environment')
+            # Jedi itself cannot deal with homedir-relative paths.
+            # On systems, where it is expected, expand the home directory.
+            import ipdb; ipdb.set_trace()
```

Comment:
```suggestion
```

I guess you used this line for testing, right?

Reply:
Oh really sorry, indeed. I was trying to debug the tests. Pushed too hastily.